src/model/document/document.ts

```diff
@@ -169,7 +179,7 @@ function useElementSelection() {
   const selectedElements = shallowReactive<Set<QuantumElement>>(new Set())
 
   function watchElement(element: QuantumElement) {
-    const stopHandle = watchImmediate(element.selected, (value) => {
+    const stopHandle = watchImmediate(element.selected, (value: Boolean) => {
```

Comment:
Use the lowercase `boolean` here. The difference is really subtle, one of them is a Boolean object and the other one is a boolean primitive.

Reply:
I did not know, thanks for finding that.